bayesian/utils.py

```diff
@@ -223,18 +223,17 @@ def search_packages_from_graph(tokens):
     # TODO query string for actual STAGE/PROD
     # g.V().has('vertex_label','Package').has('tokens','one').has('tokens','two').
     # out('has_version').valueMap('pecosystem', 'pname', 'version')).limit(5)
-    qstring = "g.V()"
-    # qstring = "g.V()"
+    qstring = ["g.V()"]
+    tkn_string = ".has('token', '{t}' )"
     for tkn in tokens:
         if tkn:
             # TODO Change qstring
-            qstring += ".has('tokens', '" + tkn + "')"
-            # qstring += ".has('alias', '" + tkn + "')"
+            qstring.append(tkn_string.format(t=tkn))
```

Comment:
I do not have any strong preference, but qstring += was more readable to me personally. LGTM so I will merge this anyways.